Lib/idlelib/idle_test/run_coverage.py

```diff
@@ -0,0 +1,135 @@
+import argparse
+import os
+import pathlib
+import subprocess
+import sys
+import textwrap
+
+
+def main():
+    idlelib_path = find_idlelib_path()
+    sources_root_path = idlelib_path.parent.parent
+    venv_path, venv_python_path = ensure_venv(sources_root_path)
+
+    module_names = sorted(get_module_names(idlelib_path))
+
+    parser = argparse.ArgumentParser(
+        description='Run tests for an idlelib module '
+                    'and collect coverage stats',
+    )
+    parser.add_argument('module', choices=module_names)
+    args = parser.parse_args()
+    module_name = args.module
+
+    os.chdir(sources_root_path)
+
+    coveragerc_path = pathlib.Path('.coveragerc')
+    orig_coverage_rc = None
+    if coveragerc_path.is_file():
+        with coveragerc_path.open('r', encoding='utf-8') as f:
+            orig_coverage_rc = f.read()
+    try:
+        with coveragerc_path.open('w', encoding='utf-8') as f:
+            f.write(textwrap.dedent('''\
+                [run]
+                branch = True
+                
```

Comment:
Trailing whitespace

Reply:
Thanks for pointing these out. I often leave these indentations in multi-line indented strings such as thins, but in this case I'll stick to pep8 and remove them. :+1:

Reply:
The reason I pointed this out is because Travis and Azure are failing due to these. Hopefully I will get a chance to look at the actual code proposed soon.